docs/running-node-manual/on.md

```diff
@@ -0,0 +1,117 @@
+# Running an Observer Node manually
+
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+## Deployment steps
+
+### 1. Put `cosmovisor` binary to `/usr/bin/`, set proper owner and execution permissions.
+
+### 2. Locate the genesis app version to genesis application version directory:
+- Create `$HOME/.dcl/cosmovisor/genesis/bin` directory.
+- Copy `dcld` binary to it, set proper owner and execution permissions.
+    Please note that execution permissions on `dcld` should be granted to all (i.e. User, Group and Others classes)
+    because cosmovisor requires execution permission on the application binary to be granted to Others class.
+
+### 3. Configure CLI:
+- `./dcld config chain-id testnet`
+- `./dcld config output json` - Output format (text/json).
+
+### 4. Initilize the node:
+
+```bash
+./dcld init "<node-name>" --chain-id "<chain-id>"
+```
+- Use `testnet` if you want to connect to the persistent Test Net
+
+### 5. (Optional) Consider enabling `state sync` in the configuration if you are joining long-running network
```

Comment:
Please provide a command

Reply:
I can't provide a single command, since it is a manual step. But I provided the link to the document  `docs/advanced/running-node-in-existing-network.md`